### PR TITLE
Coverity 1021706: Init members

### PR DESCRIPTION
**Pull request by zwoop**

### CODE REVIEW

**Comment by atsci** [comment]
clang format *successful*! https://ci.trafficserver.apache.org/job/clang-format-github/468/

**Comment by atsci** [comment]
RAT check *successful*! https://ci.trafficserver.apache.org/job/RAT-github/478/

**Comment by atsci** [comment]
FreeBSD11 build *failed*! https://ci.trafficserver.apache.org/job/freebsd-github/2161/

**Comment by atsci** [comment]
Linux build *successful*! https://ci.trafficserver.apache.org/job/linux-github/2056/

**Comment by atsci** [comment]
Intel CC build *successful*! https://ci.trafficserver.apache.org/job/icc-github/592/

**Comment by atsci** [comment]
AU check *successful*! https://ci.trafficserver.apache.org/job/autest-github/466/

**Comment by atsci** [comment]
clang-analyzer build *successful*! https://ci.trafficserver.apache.org/job/clang-analyzer-github/721/

File: iocore/cache/RamCacheLRU.cc [hunk 1]
```diff
@@ -36,8 +36,8 @@ struct RamCacheLRUEntry {
 
 struct RamCacheLRU : public RamCache {
   int64_t max_bytes = 0;
-  int64_t bytes = 0;
-  int64_t objects = 0;
+  int64_t bytes     = 0;
+  int64_t objects   = 0;
```
Comment (PSUdaemon) [inline iocore/cache/RamCacheLRU.cc]: You should rebase to remove this.

Comment (zwoop) [inline iocore/cache/RamCacheLRU.cc]: Done.

**Comment by atsci** [comment]
clang format *successful*! https://ci.trafficserver.apache.org/job/clang-format-github/484/

**Comment by atsci** [comment]
RAT check *successful*! https://ci.trafficserver.apache.org/job/RAT-github/493/

**Comment by atsci** [comment]
AU check *successful*! https://ci.trafficserver.apache.org/job/autest-github/481/

**Comment by atsci** [comment]
FreeBSD11 build *successful*! https://ci.trafficserver.apache.org/job/freebsd-github/2176/

**Comment by atsci** [comment]
Intel CC build *successful*! https://ci.trafficserver.apache.org/job/icc-github/607/

**Comment by atsci** [comment]
Linux build *successful*! https://ci.trafficserver.apache.org/job/linux-github/2072/

**Comment by atsci** [comment]
clang-analyzer build *successful*! https://ci.trafficserver.apache.org/job/clang-analyzer-github/736/

**Pull request merged**